aries-site/src/examples/components/layer/LayerFormExample.js

```diff
@@ -0,0 +1,173 @@
+import React, { useContext, useState } from 'react';
+import {
+  Button,
+  Box,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  ResponsiveContext,
+  Text,
+  TextInput,
+  TextArea,
+  Select,
+} from 'grommet';
+
+const options = ['Any', 'None', 'Any Scheduled'];
+
+export const LayerFormExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    monitorName: 'C2 Monitor',
+    desc: `Sustainable neutra echo park helvetica freegan
+    forage hashtag meditation kogi activated charcoal
+    pitchfork`,
+    monitorType: 'Rule',
+    frameworks: 'Any Scheduled',
+    severity: 'Any',
+    provider: 'Any',
+    account: 'Any',
+    ruleName: 'Any Scheduled',
+    email: 'jane@hpe.com',
+  });
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+  return (
+    <>
+      <Box align="start">
+        <Button label="Show me the Layer" onClick={onOpen} primary />
+      </Box>
+      {open && (
+        <Layer
+          position="right"
+          full={!['xsmall', 'small'].includes(size) ? 'vertical' : true}
+          modal
+          onClickOutside={onClose}
+          onEsc={onClose}
+        >
+          <Box
+            fill="vertical"
+            gap="medium"
+            overflow="auto"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+            pad="medium"
+          >
+            <Header
+              direction="row"
+              align="center"
+              as="header"
+              justify="between"
+            >
+              <Heading margin="none" level={2} size="small">
+                Add Monitor
+              </Heading>
+            </Header>
+            <Box
+              // Padding used to prevent focus from being cutoff
+              pad={{ horizontal: 'xsmall' }}
+              // overflow to allow the body to scroll with keeping the
+              // header at the top
+              overflow="auto"
+            >
+              <Form
```

Comment:
There is a lot reused from this example to LayerStickyFormExample -- is there any way that they can share the same code? but we just flag for `sticky="scrollup" in one file?

OR pull the form content from an existing form example?

Reply:
I can pull the `form` context from an existing example that's what I was thinking I did attempt to do something like flag for `sticky` it was not working on my end